### PR TITLE
Using new rad and adjust output for consumer logic

### PR DESCRIPTION
**Pull request by jameswnl**
Depending on https://github.com/ManageIQ/aiops-rad/pull/5. Will update pinned version to 0.9.1

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -7,11 +7,12 @@ name = "pypi"
 requests = ">=2.20.0"
 gunicorn = "*"
 Flask = "*"
-rad = {git = "https://github.com/ManageIQ/aiops-rad.git",ref = "v0.8.2"}
+rad = {editable = true,git = "https://github.com/parsahosseini/aiops-rad.git",ref = "isolationforest"}
```
Comment (jameswnl) [inline Pipfile]: will update this to use `v0.9.1` once the dependent `rad` PR is merged and released

File: server.py [hunk 2]
```diff
@@ -12,10 +12,11 @@ def create_application():
     """Create Flask application instance."""
     app = Flask(__name__)
     app.config['AI_SERVICE'] = os.environ.get('AI_SERVICE')
-    app.config['NEXT_MICROSERVICE_HOST'] = \
-        os.environ.get('NEXT_MICROSERVICE_HOST')
-    app.config['IF_NUM_TREES'] = int(os.environ.get('IF_NUM_TREES') or '0')
-    app.config['IF_SAMPLE_SIZE'] = int(os.environ.get('IF_SAMPLE_SIZE') or '0')
+    app.config['NEXT_SERVICE_URL'] = os.environ.get('NEXT_SERVICE_URL')
+    app.config['IF_NUM_TREES_FACTOR'] = \
+        float(os.environ.get('IF_NUM_TREES_FACTOR') or '0.2')
```
Comment (tumido) [inline server.py]: Nitpick: this should be `float(is.environ.get('IF_NUM_THEES_FACTOR', .2))`. The `get` method can provide default value for you. No need for `or`. Please fix this it this expression and in the one below

**Review by tumido — CHANGES_REQUESTED** [review]
One nitpick, otherwise looks good!

File: server.py (hunk 2 above)
Comment (jameswnl) [reply server.py]: done

File: workers.py [hunk 3]
```diff
@@ -88,6 +139,9 @@ def worker() -> None:
             sample_size,
         )
         result = isolation_forest.predict(data_frame)
+        scores = compile_scores(result)
```
Comment (tumido) [inline workers.py]: @jameswnl Can you please move these 3 variables (`scores`, `contrasts` and `charts`) directly into the output JSON? It's gonna remove 3 variables from this function and make pylama happy, so travis would be passing.

**Review by tumido — CHANGES_REQUESTED** [review]

File: workers.py (hunk 3 above)
Comment (jameswnl) [reply workers.py]: done

**Review by tumido — APPROVED** [review]
Good job! @jameswnl and @parsahosseini!  Let's make this a real deployment thing!

**Pull request merged**